index.html

```diff
@@ -1642,6 +1642,15 @@ <h2 id="OfflineAudioContext">
             </dl>
           </dd>
           <dt>
+            readonly attribute unsigned long length
```

Comment:
Oops. It's numberOfFrames in chrome's idl.  But numberOfFrames is probably more intuitive than just length.